### PR TITLE
Implement example test for hw4

### PR DESCRIPTION
**Pull request by dbernhard-0x7CD**

### CODE REVIEW

File: 04/sharedtests.ml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+open Assert
+
+
+let shared_suite : suite = 
+  [ 
+    Test ("Dbernhard tests", Dbernhard.dbernhard_tests);
+  ]
```
Comment (nicdard) [inline 04/sharedtests.ml:7]: Please keep it consistent in style (see files under 03 folder)

File: 04/dbernhard/simple_while.oat [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+
```
Comment (nicdard) [inline 04/dbernhard/simple_while.oat:2]: Can you trailing remove whitespaces? Also please add a newline at the end of the file

File: 04/dbernhard.ml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+open Assert
+
+let prefix = "./compiler-design-eth-tests/04/dbernhard/"
+
+let simple_tests = [
+  (prefix ^ "simple_while.oat", "", "10");
+]
+
+let dbernhard_tests = (Gradedtests.executed_oat_file simple_tests)
```
Comment (nicdard) [inline 04/dbernhard.ml:10]: Please keep the style as in 03 and add a newline at the end of the file

**Review by nicdard — COMMENTED** [review]
LGTM, thanks! Just minor stylistic comments.

Also please can you rewrite commit messages in order to follow the conventions we used until now? Thansk!

**Comment by dbernhard-0x7CD** [comment]
I really need to get some formatter for ocaml.

**Comment by nicdard** [comment]
> I really need to get some formatter for ocaml.

I have tried to use ocamlformat for this with dune, still the results are not that great

In any case it's ok I don't want to be to picky don't worry

**Pull request merged**